src/Generator/Generator.php

```diff
@@ -72,7 +72,14 @@ protected function renderFile(
         $flag = null
     ) {
         if (!is_dir(dirname($target))) {
-            mkdir(dirname($target), 0777, true);
+            if(!mkdir(dirname($target), 0777, true)){
+                throw new \InvalidArgumentException(
+                    sprintf(
+                        'Path "%s" is invalid. You need to provide a valid path.',
+                        $parameters["path"]
```

Comment:
Is parameters["path"] always set? Wouldn't it be safer to use `dirname($target)` here too? Maybe putting that in a variable at the start of the function, since it's used multiple times.

Reply:
@harold20 did you see this comment?

Reply:
I already committed that recommendation @jmolivas and thanks @ao2. I had not seen that.